### DISCUSSION TITLE
Select (multiple) loading options in an odd behavior.

### DISCUSSION THREAD

**Discussion by NomisGnos** [filamentphp/filament#5255] [Help]
I have a select field that isn't loading the options to my liking. I am certain I am missing something but I cannot find the answer.  I have a RoleResource that has this in the form function:

`Select::make('permission')
      ->multiple()
      ->relationship('permissions', 'name')
      ->preload(true)
      ->label(strval(__('Permissions'))),`

This relationship is attached to Role.  When you first create a role, loads fine. You can select an option and the option "disappears" from the selection because it was selected.  After you save it and edit it again,  the entire list of options appear even though I already had some of them selected.

For example:
![image](https://user-images.githubusercontent.com/4072574/207664473-5471bba9-4b08-4e28-919f-4f73813cff8b.png)

SOMETIMES if I select any of the options that weren't selected, and close the list from the select box and reselect it, it shows me the correct listing.
![image](https://user-images.githubusercontent.com/4072574/207665001-e76b2d02-e4a5-4af8-ad95-b4202a71225d.png)
Not always...very random.

If I remove "preload(true)" it gives me this and start the search with a "cr" it gives me duplicates.
![image](https://user-images.githubusercontent.com/4072574/207665537-acc577b2-b10c-4e82-8981-09ba6326e1ae.png)

**Comment by danharrin** [comment]
I think this is https://github.com/filamentphp/filament/issues/2363
